frontend/public/components/nav/section.tsx

```diff
@@ -40,10 +46,10 @@ const mergePluginChild = (
 };
 
 export const NavSection = connect(navSectionStateToProps)(
-  withExtensions(
-    isNavItem,
-    isPerspective,
-  )(
+  withExtensions<NavSectionExtensionProps>({
```

Comment:
As discussed before, this type param shouldn't be necessary, but I'd do that as a follow-up improvement.

That said, the `withExtensions` HOC is currently used on a single place, hopefully that number stays low.